.gitlab-ci.yml

```diff
@@ -32,49 +32,64 @@ RPM:
   parallel:
     matrix:
       - RUNNER:
-          - aws/fedora-32-x86_64
+#          - aws/fedora-32-x86_64
           - aws/fedora-33-x86_64
-          - aws/fedora-33-aarch64
-          - aws/fedora-34-x86_64
-          - aws/fedora-34-aarch64
+#          - aws/fedora-33-aarch64
+#          - aws/fedora-34-x86_64
+#          - aws/fedora-34-aarch64
           - aws/rhel-8-x86_64
-          - aws/rhel-8-aarch64
-          - aws/centos-stream-8-x86_64
-          - aws/centos-stream-8-aarch64
-      - RUNNER:
-          - aws/rhel-8.4-x86_64
-        INTERNAL_NETWORK: ["true"]
-
-Base:
-  stage: test
-  extends: .terraform
-  variables:
-    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
-  script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/base_tests.sh
-  parallel:
-    matrix:
-      - RUNNER:
-          - aws/fedora-32-x86_64
-          - aws/fedora-33-x86_64
-          - aws/fedora-33-aarch64
-          - aws/rhel-8-x86_64
-          - aws/rhel-8-aarch64
-          - aws/centos-stream-8-x86_64
-          - aws/centos-stream-8-aarch64
-      - RUNNER:
-          - aws/rhel-8.4-x86_64
-        INTERNAL_NETWORK: ["true"]
-
-OSTree:
+#          - aws/rhel-8-aarch64
+#          - aws/centos-stream-8-x86_64
+#          - aws/centos-stream-8-aarch64
+#      - RUNNER:
+#          - aws/rhel-8.4-x86_64
+#        INTERNAL_NETWORK: ["true"]
+#
+#Base:
+#  stage: test
+#  extends: .terraform
+#  variables:
+#    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
+#  script:
+#    - schutzbot/deploy.sh
+#    - /usr/libexec/tests/osbuild-composer/base_tests.sh
+#  parallel:
+#    matrix:
+#      - RUNNER:
+#          - aws/fedora-32-x86_64
+#          - aws/fedora-33-x86_64
+#          - aws/fedora-33-aarch64
+#          - aws/rhel-8-x86_64
+#          - aws/rhel-8-aarch64
+#          - aws/centos-stream-8-x86_64
+#          - aws/centos-stream-8-aarch64
+#      - RUNNER:
+#          - aws/rhel-8.4-x86_64
+#        INTERNAL_NETWORK: ["true"]
+#
+#OSTree:
+#  stage: test
+#  extends: .terraform
+#  variables:
+#    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
+#  script:
+#    - schutzbot/deploy.sh
+#    - /usr/libexec/tests/osbuild-composer/ostree.sh
+#  parallel:
+#    matrix:
+#      - RUNNER:
+#          - openstack/fedora-33-x86_64
+#          - openstack/rhel-8-x86_64
+#
+# try just libvirt first
+Integration:
   stage: test
   extends: .terraform
   variables:
     EXTRA_REPO_PATH_SEGMENT: "gitlab/"
```

Comment:
This will need all of the environment variables from Jenkinsfile (note, TEST_TYPE doesn't seem to be used, DISTRO_CODE can be removed in favor of /etc/os-release). 

@ondrejbudai do you mind configuring some of the credentials in GitLab or tell us which ones are already configured ?

Some links I've found on the topic:
https://docs.gitlab.com/ee/ci/variables/#project-cicd-variables (Project or Group variables with masking)
https://docs.gitlab.com/ee/ci/secrets/
https://secrethub.io/docs/guides/gitlabci/

Reply:
Current job fails b/c of the missing DISTRO_CODE variable.

Reply:
You can see all the configured credentials in the GitLab UI. I think we cannot remove `DISTRO_CODE` that easily because it's used to distinguish RHEL 9 tests (they run on RHEL 8).

Reply:
> Current job fails b/c of the missing DISTRO_CODE variable.

Yeah, I'm currently trying to figure out the best way to define variables specific to each runner like we had in Jenkinsfile for each stage...

Reply:
IMO the should either be specific to the job we're running (base vs integration, etc) or common for all jobs. There's no need for specific variables per runner b/c the test jobs on different OS/arch combo still need the same env variables.

Reply:
Yeah it seems DISTRO_CODE is the only one that is runner specific... It's not used in that many places so it could be replaced easily with something like ID+VERSION_ID from /etc/os-release like you suggested.. Defining variables per groups of jobs should be easy then, there is already one defined in the .gitlab-ci.yaml and secrets can be added in settings in the gitlab repository...